### PR TITLE
Update session metadata

### PR DESCRIPTION
**Pull request by AniketSK**

### CODE REVIEW

File: docs/android/v2/how--to-guides/interact-with-room/room/session-metadata.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Session metadata is a shared state that is accessible to all participants in a room. It can be used for features such as pinning text or highlighting a particular participant. The session metadata is retained throughout the duration of a session and is cleared once all participants have left the room.
+
+## Differences between Session Metadata and Peer Metadata
+
+While [peer metadata](https://www.100ms.live/docs/android/v2/how--to-guides/interact-with-room/peer/peer-metadata-update) is associated with individual peers and each peer can have their own metadata, session metadata remains the same for every peer in the room.
+
+## How to store shared data with 100ms SDK
+
+To update the value of the session metadata for the current room session, call `setSessionMetadata` method on SDK instance. In the following example, the "test" string will be saved in the session metadata and accessible to all participants in the room as a shared state.
```
Comment (arjunattam) [inline docs/android/v2/how--to-guides/interact-with-room/room/session-metadata.mdx:14]: is it `setSessionMetadata` or `setSessionMetaData` - seeing inconsistent casing between text and code snippet

Comment (arjunattam) [reply docs/android/v2/how--to-guides/interact-with-room/room/session-metadata.mdx:14]: same for the getter method below

File: docs/android/v2/how--to-guides/interact-with-room/room/session-metadata.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    @Override
+    public void onError(@NonNull HMSException e) {
+    }
+});
+
+```
+
+</Tab>
+
+## Limitations and Alternatives in the Beta Release
```
Comment (arjunattam) [inline docs/android/v2/how--to-guides/interact-with-room/room/session-metadata.mdx:94]: are you planning to propose any "Alternatives" also (e.g. polling)?

Comment (arjunattam) [reply docs/android/v2/how--to-guides/interact-with-room/room/session-metadata.mdx:94]: just saw the further section - which answers this. can the alternative be a lower-level heading that fits inside this section?

**Review by arjunattam — APPROVED** [review]

File: docs/android/v2/how--to-guides/interact-with-room/room/session-metadata.mdx (hunk 2 above)
Comment (AniketSK) [reply docs/android/v2/how--to-guides/interact-with-room/room/session-metadata.mdx]: Split it kind of

**Comment by arjunattam** [comment]
we introduced a " in the title with the latest changes. can we fix that @AniketSK ?

**Pull request merged**